File: Week 1/mario.c
// Mario Less

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int answer = 0;
    while (answer > 8 || answer < 1)
    {
        answer = get_int("Height? ");
    }

    int height = answer;

    for (int i = 1; i <= height; i++)
    {
        for (int j = height - i; j > 0; j--)
        {
            printf(" ");
        }

        for (int k = 0; k < i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
    return 0;
}

File: cs50x - Introduction to Computer Science/week6-python/mario.py
# Mario More

from cs50 import get_int


def main():
    height = get_height()
    for i in range(1, height + 1):
        print(" " * (height - i), end="")
        print("#" * i, end="")
        print("  ", end="")
        print("#" * i)


def get_height():
    while True:
        height = get_int("Height: ")
        if height >= 1 and height <= 8:
            return height


if __name__ == "__main__":
    main()
